tabs/src/components/app/views/runner/Request.jsx

```diff
@@ -45,41 +48,54 @@ export function Request(props) {
         ],
     };
 
+    const deleteButton = () => <Button
+        tabIndex={-1}
+        icon={<TrashCanIcon className="button-icon" />}
+        circular
+        text
+        iconOnly
+        aria-label="delete"
```

Comment:
the aria label needs to be localized as well!

Reply:
https://github.com/LokiLabs/microsoft-graph-explorer-teams-app/pull/57/commits/dae8f359fc4851554f2f48601fa647dcb83de88b